metasmoke.py

```diff
@@ -78,7 +80,13 @@ def init_websocket():
                 GlobalVars.metasmoke_failures += 1
                 log('error', "Couldn't bind to MS websocket")
                 if not has_succeeded:
-                    break
+                    failed_connection_attempts += 1
+                    if failed_connection_attempts > MAX_MS_WEBSOCKET_RETRIES:
+                        log('warning', "Cannot initiate MS websocket. metasmoke_ws_t is now dead.")
+                        break
```

Comment:
Even though it wasn't in the original, it would be good to have a

    chatcommunicate.tell_rooms_with("debug", ...

here prior to the `break`, so that we're informed the current instance won't be connected to MS (and won't continue to try to connect to MS).

As I'm thinking about it, it might be better to not limit the number of attempts to connect (e.g. so SD auto-connects when SD is rebooted when MS is down for more than 50s; as it is and was SD would need to be manually rebooted). I'm not recalling the use-case of why we're specifically not doing so when MS is enabled. I'd have to look at the commit history for this file to see what might be explained there.